Lib/idlelib/editor.py

```diff
@@ -1202,12 +1202,7 @@ def load_extension(self, name):
         if keydefs:
             self.apply_bindings(keydefs)
             for vevent in keydefs:
-                methodname = vevent.replace("-", "_")
-                while methodname[:1] == '<':
-                    methodname = methodname[1:]
-                while methodname[-1:] == '>':
-                    methodname = methodname[:-1]
-                methodname = methodname + "_event"
+                methodname = vevent.lstrip("<").replace("-", "_").rstrip(">") + "_event"
```

Comment:
Is it even necessary to separate the strip of `<` and `>`?  Could we just use `vevent.strip('<>')`?
```suggestion
                methodname = vevent.strip("<>").replace("-", "_") + "_event"
```

Reply:
What if people name their extensions like >something<?

Reply:
I don't know about anyone else, but I would never trust an extension author who named a method `>something<_event` and jumped through the hoops to make that work :).  I'm not sure it even *could* work even if they *did* jump through those hoops; I'm pretty sure that would break somewhere in Tcl/Tk.